python_cli/scanner.py

```diff
@@ -60,7 +61,7 @@ def main():
     args = aparse.parse_args()
 
     global hw
-    hw = make_sniffle_hw(args.serport)
+    hw = make_sniffle_hw(serport=args.serport,baudrate=args.baudrate)
```

Comment:
Missing a space after the comma, after: `...(args.serport,`

Reply:
Solved.